assets/js/phoenix_live_view/dom_patch.js

```diff
@@ -61,7 +66,9 @@ export default class DOMPatch {
   }
 
   markPrunableContentForRemoval(){
-    DOM.all(this.container, "[phx-update=append] > *, [phx-update=prepend] > *", el => {
+    let phxUpdate = this.liveSocket.binding(PHX_UPDATE)
+    DOM.all(this.container, `[${phxUpdate}=${PHX_STREAM}]`, el => el.innerHTML = "")
```

Comment:
For consistency with 71
```suggestion
    DOM.all(this.container, `[${phxUpdate}=${PHX_STREAM}]`, el => {
      el.innerHTML = ""
    })
```